src/utils.rs

```diff
@@ -5,23 +5,25 @@ use teloxide::Bot;
 use std::collections::HashMap;
 use std::sync::{Arc, Mutex};
 
+static FORMAT_TEXT: &str = "Оберните код в теги: 3 символа ` до и после кода \
```

Comment:
Since these messages are related to specific functions (and then should have smaller scopes) and there is no performance difference between in and out variants - please move texts back to the functions.